lib/Doctrine/DBAL/Migrations/Provider/LazySchemaDiffProvider.php

```diff
@@ -41,6 +41,16 @@ public function __construct(LazyLoadingValueHolderFactory $proxyFactory, SchemaD
 
     public static function fromDefaultProxyFacyoryConfiguration(SchemaDiffProviderInterface $originalSchemaManipulator)
     {
+        trigger_error(
+            sprintf('Function deprecated. Use %s::fromDefaultProxyFactoryConfiguration() instead', self::class),
```

Comment:
you should mention the deprecation function in the message, to make the warning easier to understand